src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -59,7 +59,8 @@ See the LICENSE file in the project root for more information.
     <NativeBinaryExt Condition="'$(NativeCodeGen)' == 'wasm'">.html</NativeBinaryExt>
 
     <ExportsFileExt Condition="'$(OutputType)' !=  'Exe' and '$(TargetOS)' == 'Windows_NT' and '$(NativeLib)' == 'Shared'">.def</ExportsFileExt>
-    <ExportsFileExt Condition="'$(OutputType)' !=  'Exe' and '$(TargetOS)' != 'Windows_NT' and '$(NativeLib)' == 'Shared'">.exports</ExportsFileExt>
+    <ExportsFileExt Condition="'$(OutputType)' !=  'Exe' and '$(TargetOS)' == 'OSX' and '$(NativeLib)' == 'Shared'">.exports</ExportsFileExt>
+    <ExportsFileExt Condition="'$(OutputType)' !=  'Exe' and '$(TargetOS)' != 'Windows_NT' and '$(TargetOS)' != 'OSX' and '$(NativeLib)' == 'Shared'">.map</ExportsFileExt>
```

Comment:
Nit: I am not sure how standard is the `.map` suffix for version scripts. I have done a quick search https://github.com/search?q=--version-script&type=Code and it is all over the place.

It may be ok to use `.exports` for them to keep things simpler. It is what the CoreCLR build does.

Reply:
.map is used at https://www.gnu.org/software/gnulib/manual/html_node/LD-Version-Scripts.html
I think that there is no problem with use .exports, really it's just a flat file.